src/components/SecondaryPanes/Frames/tests/Frames.js

```diff
@@ -53,6 +54,31 @@ describe("Frames", () => {
     expect(showMore.props.className).toEqual("show-more");
   });
 
+  describe("Blackboxed Frames", () => {
+    fit("filters blackboxed frames", () => {
```

Comment:
left over `fit` 

(in case anyone is reading this and asking "what the heck is fit" its a shorter way of writing `it.only`, and forces jest to run just one test in the file)